algorithm/cgal/computeOrientedBoundingBox.js

```diff
@@ -6,25 +6,29 @@ export const computeOrientedBoundingBox = (inputs) => {
   if (inputs.length === 0) {
     return;
   }
-  return withCgalGeometry(inputs, (cgalGeometry, g) => {
-    const status = g.ComputeOrientedBoundingBox(cgalGeometry);
-    // This adds segments with twelve entries: length, depth, height, ...
-    switch (status) {
-      case STATUS_ZERO_THICKNESS:
-        throw new ErrorZeroThickness(
-          'Zero thickness produced by computeOrientedBoundingBox'
-        );
-      case STATUS_OK:
-        return fromCgalGeometry(
-          cgalGeometry,
-          inputs,
-          inputs.length + 1,
-          inputs.length
-        );
-      case STATUS_EMPTY:
-        return;
-      default:
-        throw new Error(`Unexpected status ${status}`);
+  return withCgalGeometry(
+    'computeOrientedBoundingBox',
+    inputs,
+    (cgalGeometry, g) => {
+      const status = g.ComputeOrientedBoundingBox(cgalGeometry);
```

Comment:
My brain read this as "Computer oriented bounding box" the first time through and I was like "What is the computer frame of reference? Doesn't it move with the camera? 🙃